tests/drivers/tee/optee/src/main.c

```diff
@@ -276,34 +276,44 @@ ZTEST(optee_test_suite, test_reg_unreg)
 	struct tee_shm *shm = NULL;
 	const struct device *const dev = DEVICE_DT_GET_ONE(linaro_optee_tz);
 
+	t_call.num = 0;
+	t_call.smc_cb = normal_call;
```

Comment:
It will be really nice to test for huge buffers (more than 512 pages) to ensure that pagelists are working correctly.